C/CUTEst/build_tarballs.jl

```diff
@@ -0,0 +1,131 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CUTEst"
+version = v"0.1.0"
+
+# Collection of sources required to build ThinASLBuilder
+sources = [
+    "https://github.com/ralna/ARCHDefs/archive/v2.0.3x.tar.gz" =>
+    "6583e27f84338447767bbdf4335514c8836ae4ad54f5e66280307e8b57189cff",
+    "https://github.com/ralna/SIFDecode/archive/v2.0.2.tar.gz" =>
+    "72936047dbea3695af6a15f9193412c22c119a3bc21d925750abf610c5df181b",
+    "https://github.com/ralna/CUTEst/archive/v2.0.3.tar.gz" =>
+    "d21a65c975302296f9856c09034cf46edc5da34b6efd96eed6cc94af6d2c8a55",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+echo "building for ${target}"
```

Comment:
Note that the very first line of the output of `build_tarballs()` is something like
```
[ Info: Building for x86_64-linux-gnu
```
E.g., see https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=1048&view=logs&j=c5df49c6-f935-5d99-d080-4aec88ea536b&t=1d591ce1-f371-5287-c2bd-9834269dd452&l=13